README.md

```diff
@@ -205,12 +221,6 @@ telescope.setup {
 
 ```
 
-### Known Limitations
-
-For files not already in your history, smart-open uses ripgrep for scanning the current directory.  (The command is roughly: `rg --files --glob-case-insensitive --hidden --ignore-file=<cwd>/.ff-ignore -g <ignore_patterns...>`).
-
-As a result, files added to git, _but also ignored by git_, will not be included.  While not common, this is something that git allows. If this becomes a problem you can work around it by either changing your git ignore patterns, editing the file in neovim in some other way, (thereby adding it to the history), or by using ripgrep's `.ignore` file for overriding git.
```

Comment:
afaik, this is still a limitation.  Ripgrep will not list a file that is added to git if it also matches a .gitignore pattern.  (I believe `git ls-files` does the right thing here, and would list such files.)